src/Form/matching-config/selectors-css.js

```diff
@@ -104,6 +104,8 @@ const username = [
     `input[id="nw_username" i]`,
     // https://appleid.apple.com/es/sign-in (needed for all languages)
     `input[can-field="accountName"]`,
+    // https://www.fullcolor.com/customer/account/login/
+    `input[name="login[username]"]`,
```

Comment:
Some times we had to revert to fixing using straight up css selectors. We try to do it only when they are unambiguous and could be generic enough. Or sometimes we just add them to fix the bug at hand, when the site is high-profile or we have good reason.